app/views/templates/credit_note.slim

```diff
@@ -429,7 +429,7 @@ html
                 tr
                   td.body-1
                     - if item.fee.true_up_parent_fee_id?
-                      | #{I18n.t('invoice.true_up_metric', metric: item.fee.invoice_name)}
+                      | #{I18n.t('invoice.true_up_metric', metric: item.fee.true_up_parent_fee.invoice_name)}
```

Comment:
This change seems not to be related to this PR objective, does it?
Is that only a rebase issue maybe?

Reply:
discussed together, it's expected